certbot-nginx/certbot_nginx/configurator.py

```diff
@@ -62,6 +62,14 @@
     ['#', " } # managed by Certbot"],
 ]
 
+PREVIOUS_SSL_OPTIONS_HASHES = [
+    '0f81093a1465e3d4eaa8b0c14e77b2a2e93568b0fc1351c2b87893a95f0de87c',
```

Comment:
Right, but I don't think you have a hash of that version of the file. It looks like the oldest hash of the file you have is from commit 6b639679e8df0c860e16f571dc831bc339132623 where we changed the name of the SSL session cache.